### PR TITLE
fix linux limit-the-wget-download

### PR DESCRIPTION
**Pull request by Stefan-Stojanovic**
Mark issue after merging
createdAt Sep 12, 2019 11:24

### CODE REVIEW

**Comment by enki-hq** [comment]
Rule insight:author failed for files: 
  - linux/networking-and-security/webscraping/limit-the-wget-download-rate.md

**Comment by enki-hq** [comment]
Rule insight:author failed for files: 
  - linux/networking-and-security/webscraping/limit-the-wget-download-rate.md

**Comment by enki-hq** [comment]
Rule insight:author failed for files: 
  - linux/networking-and-security/webscraping/limit-the-wget-download-rate.md

**Comment by enki-hq** [comment]
### Insight previews:
- [ ] [enkidevs/curriculum/linux/networking-and-security/webscraping/limit-the-wget-download-rate.md](https://app.enkipro.com/#/public/preview-insight/enkidevs/curriculum/linux/networking-and-security/webscraping/limit-the-wget-download-rate.md?ref=bd6b878b7716cc540b84d83ecb8f0c3cbda97b38)

File: linux/networking-and-security/webscraping/limit-the-wget-download-rate.md [hunk 1]
```diff
@@ -14,23 +8,6 @@ type: normal
...
-
-  - wget
-
-  - rate
-
-  - limit
-
-  - terminal
-
-  - download
```
Comment (kapnobatai137) [inline linux/networking-and-security/webscraping/limit-the-wget-download-rate.md:29]: no need to remove tags. although they are optional (and not used currently), we might use them in the future for search features

Comment (Stefan-Stojanovic) [reply linux/networking-and-security/webscraping/limit-the-wget-download-rate.md:29]: Ok, ill add them back

Comment (Stefan-Stojanovic) [reply linux/networking-and-security/webscraping/limit-the-wget-download-rate.md]: Done

**Pull request merged**